<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajout_contact</class>
 <widget class="QDialog" name="ajout_contact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="rbPro">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Professionel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbPrive">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Privé</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="w_prive" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>251</width>
     <height>89</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridPrive">
    <item row="0" column="1">
     <widget class="QDateEdit" name="dateNaissance">
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lblDate">
      <property name="text">
       <string>Date de Naissance :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="w_pro" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridPro">
    <item row="0" column="0">
     <widget class="QLabel" name="lblNomE">
      <property name="text">
       <string>Nom Entreprise :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="leNemE"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblMail">
      <property name="text">
       <string>Mail : </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="leMail"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblNom">
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="leNom"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblPrenom">
      <property name="text">
       <string>Prenom :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lePrenom"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblGenre">
      <property name="text">
       <string>Genre :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="leGenre"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>378</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridAdresse">
    <item row="2" column="0">
     <widget class="QLabel" name="lblRue">
      <property name="text">
       <string>Rue :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="lblComp">
      <property name="text">
       <string>Compément :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="leCom"/>
    </item>
    <item row="1" column="5">
     <widget class="QLineEdit" name="leCP"/>
    </item>
    <item row="0" column="2" colspan="3">
     <widget class="QLabel" name="lblAdresse">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color : black;</string>
      </property>
      <property name="text">
       <string>Adresse : </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblNum">
      <property name="text">
       <string>Numéro :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QLabel" name="lblCp">
      <property name="text">
       <string>Code Postal :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLineEdit" name="leVille"/>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QLabel" name="lblVille">
      <property name="text">
       <string>Ville :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="leRue"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="leNum"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
